src/openzaak/config/setup_configuration/steps/selectielijst.py

```diff
@@ -0,0 +1,54 @@
+# SPDX-License-Identifier: EUPL-1.2
+# Copyright (C) 2024 Dimpact
+
+from django_setup_configuration.configuration import BaseConfigurationStep
+from zgw_consumers.constants import APITypes, AuthTypes
+from zgw_consumers.models import Service
+
+from openzaak.config.setup_configuration.models import SelectielijstAPIConfig
+from openzaak.selectielijst.models import ReferentieLijstConfig
+
+
+class SelectielijstAPIConfigurationStep(BaseConfigurationStep[SelectielijstAPIConfig]):
+    """
+    Configure the Open Zaak client to request Selectielijst API
+
+    1. Create service for Selectielijst API
+    2. Set up configuration for Selectielijst API client
+
+    Normal mode doesn't change the allowed and default years
+    If they are changed, run this command with 'overwrite' flag
```

Comment:
This no longer applies in the new version of django-setup-configuration, so I think you can remove these lines